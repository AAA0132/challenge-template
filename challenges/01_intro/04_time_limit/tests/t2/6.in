444218712392847235365
