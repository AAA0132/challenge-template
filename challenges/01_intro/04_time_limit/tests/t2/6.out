98665132219979564983987734403158276959295
